tap_bigcommerce/schemas/orders.json

```diff
@@ -442,7 +442,7 @@
             }
           },
           "external_id": {
-            "$ref": "type-integer.json"
+            "$ref": "type-string.json"
```

Comment:
I did some digging, and it looks like this specific issue isn't widespread. Given that this is pretty impactful, since the tap seems to silently drop all data that doesn't match the schema, I'm willing to merge it, as long as the change is backwards compatible. In this case, that would mean making the schema a fallback from most specific to least specific, e.g., something like `{"type": ["integer", "string", "null"]}`.

If you could make that change, and show redacted logs of the tap extracting data successfully (to ensure the JSON schema syntax is correct), then I would be willing to merge this change.

Reply:
@dmosorast changes made to row 444 with fallback